### PR TITLE
Feature 1486523: Rename Accessibility.Insights runtime to Axe.Windows

### PR DESCRIPTION
**Pull request by RobGallo**
#### Describe the change

This is the main rename for components which will soon be part of Axe.Windows.

No logical changes, only renames, were made in this PR.

### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights.Actions/Actions/DataManager.cs [hunk 1]
```diff
@@ -135,7 +135,7 @@ internal A11yElement GetA11yElement(Guid ecId, int eId)
         /// <summary>
         /// default instance name constant
         /// </summary>
-        const string DefaultInstanceName = "AccessibilityInsightsDefault";
+        const string DefaultInstanceName = "Axe.WindowsDefault";
```
Comment (karanbirsingh) [inline src/AccessibilityInsights.Actions/Actions/DataManager.cs]: Will this change have any side effects?

File: src/AccessibilityInsights.Core/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("AccessibilityInsights.Core")]
+[assembly: AssemblyTitle("Axe.Windows.Core")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("AccessibilityInsights.Core")]
+[assembly: AssemblyProduct("Axe.Windows.Core")]
 [assembly: AssemblyCopyright("Copyright �  2017")]
```
Comment (DaveTryon) [inline src/AccessibilityInsights.Core/Properties/AssemblyInfo.cs]: Looks like the copyright character is incorrect (was incorrect before, is still incorrect)

File: src/AccessibilityInsights.CoreTests/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -6,11 +6,11 @@
...
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("AccessibilityInsights.CoreTests")]
+[assembly: AssemblyTitle("Axe.Windows.CoreTests")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("AccessibilityInsights.CoreTests")]
+[assembly: AssemblyProduct("Axe.Windows.CoreTests")]
 [assembly: AssemblyCopyright("Copyright �  2017")]
```
Comment (DaveTryon) [inline src/AccessibilityInsights.CoreTests/Properties/AssemblyInfo.cs]: Copyright symbol is invalid (was before, still is)

File: src/AccessibilityInsights.RuleSelection/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -8,11 +8,11 @@
...
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("AccessibilityInsights.Common")]
+[assembly: AssemblyTitle("Axe.Windows.RuleSelection")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("AccessibilityInsights.Common")]
+[assembly: AssemblyProduct("Axe.Windows.RuleSelection")]
 [assembly: AssemblyCopyright("Copyright �  2018")]
```
Comment (DaveTryon) [inline src/AccessibilityInsights.RuleSelection/Properties/AssemblyInfo.cs]: Copyright symbol is invalid (was before, still is)

File: src/AccessibilityInsights.DesktopTests/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -6,11 +6,11 @@
...
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("AccessibilityInsights.DesktopTests")]
+[assembly: AssemblyTitle("Axe.Windows.DesktopTests")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("AccessibilityInsights.DesktopTests")]
+[assembly: AssemblyProduct("Axe.Windows.DesktopTests")]
 [assembly: AssemblyCopyright("Copyright �  2017")]
```
Comment (DaveTryon) [inline src/AccessibilityInsights.DesktopTests/Properties/AssemblyInfo.cs]: Copyright symbol is invalid (was before, still is)

**Comment by neocln** [comment]
I couldn't find out MSI/product.wix file update. 
without updating it, it will fail with MSI generation since many assembly names are changed. 
please make sure that it is changed.

**Comment by karanbirsingh** [comment]
> I couldn't find out MSI/product.wix file update.
> without updating it, it will fail with MSI generation since many assembly names are changed.
> please make sure that the Wix file is updated with the correct names.

@neocln please see this file: https://github.com/Microsoft/accessibility-insights-windows/blob/f5bfce749ba9aee2164400c1aa8e5dba6a26b159/src/MSI/Product.wxs

**Comment by neocln** [comment]
> > I couldn't find out MSI/product.wix file update.
> > without updating it, it will fail with MSI generation since many assembly names are changed.
> > please make sure that the Wix file is updated with the correct names.
> 
> @neocln please see this file: https://github.com/Microsoft/accessibility-insights-windows/blob/f5bfce749ba9aee2164400c1aa8e5dba6a26b159/src/MSI/Product.wxs

thanks for correcting me. I was fooled by Codeflow since it shows me the list of files from a single commit than all commits. I can see the file and it looks good.

**Pull request merged**